wordpress-mobile/WordPress-iOS
# Notifications Refresh

### Details:
In this PR we're:

1. Mapping a (patched up) version of WordPressShared, which allows us to get scaled fonts with the actual specified traits, rather than always regular.
2. Adjusting Notifications Font Mapping (**bold** for Users, Comments and Posts)
3. Slightly adjusting the Notifications TableView Cell
4. Adjusting the Notifications TableView Header's height

Closes #13151

**Note:** New font size is pending!

### To test:
1. Log into your WordPress.com account
2. Press over the Notifications Tab

- [ ] Verify that Users / Post Titles / Comments are now displayed in bold
- [ ] Verify that the new metrics match the mockup 


PR submission checklist:

---

### Screenshot:

**Yes**, blur on purpose, here and there. But the (new) metrics can be appreciated!

![Notifications](https://user-images.githubusercontent.com/1195260/71269941-2fc38800-232f-11ea-871c-146f5e072b75.png)
